demos/serverless/src/handlers.js

```diff
@@ -14,16 +14,44 @@ const ddb = new AWS.DynamoDB();
 // the meeting is hosted in.
 const chime = new AWS.Chime({ region: 'us-east-1' });
 
+const endpoint = process.env.CHIME_ENDPOINT;
+const globalEndpoint = "https://service.chime.aws.amazon.com";
 // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
-chime.endpoint = new AWS.Endpoint(process.env.CHIME_ENDPOINT);
+chime.endpoint = new AWS.Endpoint(endpoint);
 
-const chimeRegional = new AWS.ChimeSDKMeetings({ region: 'us-east-1' });
+
+const chimeMeetingsRegionalEndPoints = getRegionalEndpoints();
+
+
+function checkValidControlRegion(controlRegion){
+  return (controlRegion == 'us-east-1' || controlRegion == 'us-west-2' || controlRegion == 'eu-central-1' || controlRegion == 'ap-southeast-1');
+}
+
+function getRegionalEndpoints(){
+  let regionalEndpoints = {
+    "us-east-1": process.env.CHIME_MEETINGS_IAD_ENDPOINT,
```

Comment:
You need single environment variable CHIME_MEETINGS_ENDPOINT set to a specific value.

Reply:
We need all of them as we need to switch between them based upon url parameter `controlRegion`. So in case controlRegion is us-east-1 we need IAD endpoint and if controlRegion is us-west-2 we need pdx endpoint